Makefile

```diff
@@ -11,6 +11,10 @@ test-unit:
 test-docker:
 	@./tests/docker/test.sh
 
+.PHONY: test-integration
+test-integration:
```

Comment:
When we add more integration tests we can make it automatic to find the integration tests.